src/System.IO.FileSystem/src/System/IO/Win32FileStreamAsyncResult.cs

```diff
@@ -1,92 +1,49 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using Microsoft.Win32;
 using Microsoft.Win32.SafeHandles;
 using System.Security;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Runtime.InteropServices;
-using System.Runtime.CompilerServices;
-using System.Globalization;
-using System.Runtime.Versioning;
 using System.Diagnostics;
 
 namespace System.IO
 {
     internal partial class Win32FileStream
     {
+#if USE_OVERLAPPED
         // This is an internal object implementing IAsyncResult with fields
         // for all of the relevant data necessary to complete the IO operation.
         // This is used by AsyncFSCallback and all of the async methods.
         unsafe private sealed class FileStreamAsyncResult : IAsyncResult
```

Comment:
This will fixed in the next couple of weeks. We're working on a new overlapped API that works for .NET Native and everywhere else, where this isn't a requirement,